src/i18n/DE/grid-resources.ts

```diff
@@ -0,0 +1,132 @@
+import { IGridResourceStrings } from 'igniteui-angular';
+
+// exported below as re-cast to create declaration type with expanded properties
+const GridResourceStringsDE_: ExpandRequire<IGridResourceStrings> = {
+  igx_grid_groupByArea_message: 'Lege eine Spaltenüberschrift hier ab, um danach zu sortieren.',
+  igx_grid_emptyFilteredGrid_message: 'Keine Einträge gefunden.',
+  igx_grid_emptyGrid_message: 'Grid hat keine Daten.',
```

Comment:
@klausloeffelmann, how would you translate "grid" to German?

Reply:
Datentabelle ?

Reply:
Datagrid/Grid is often used in German as a loan-word, which is why I used it.